main.tf

```diff
@@ -2,6 +2,4 @@
 // TODO: define locals at the top of main.tf
 
 locals {
-  a = "a"
```

Comment:
was this unused?

Reply:
Yes, tflint didn't like it. It was just there as an example/placeholder.